templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -55,11 +54,12 @@ Metadata:
           - ESSnapshotId
           - HomeVolumeSnapshotId
           - HomeDeleteOnTermination
-          - StartCollectd
+          - DeploymentAutomationRepository
+          - DeploymentAutomationBranch
+          - DeploymentAutomationPlaybook
+          - DeploymentAutomationKeyName
 
     ParameterLabels:
-      AMIOpts:
```

Comment:
Do we know how the `AMIOpts` were used before? Was it only for Server AMI?

Reply:
No ideal. It's not used in the old AMIs, just a passing reference in some test code.